docs/UsingSeal.md

```diff
@@ -214,55 +221,59 @@ To decrypt an encrypted object in a Move package, follow these steps:
 
 You can use the TypeScript SDK to build a transaction that calls Seal’s on-chain decryption functions. 
 
-Before encrypting, make the public keys available on-chain so clients can verify them if needed. For each public key, create a corresponding Move object:
+Before encryption, the content creator may create a list of `PublicKey` Move objects containing the key server object ID and its public key. Clients can verify them if needed. 
+
 ```typescript
 const publicKey = tx.moveCall({
   target: `${SEAL_PACKAGE_ID}::bf_hmac_encryption::new_public_key`,
   arguments: [
     tx.pure.address(keyserverId),
-    tx.pure.vector("u8", Array.from(publicKeyBytes))
+    tx.pure.vector("u8", Array.from(publicKeyBytes)) // stored in the desired key server object's dynamic object field `pk`. 
   ],
 });
+const allPublicKeys = [publicKey]; // can be many
 ```
 
-Assume you have:
+Now lient can first verify the keys, then decrypt using the Move functions. 
 
-- `encryptedBytes`: a BCS-serialized encrypted object,
-- `txBytes`: a transaction block that calls a `seal_approve*` function (see [Decryption](#decryption)).
-- `allPublicKeys`: an array of Move objects for all public keys in the encryption,
-- `correspondingPublicKeys`: the public keys that correspond to the derived keys.
+Client should have:
+
+- `encryptedBytes`: the BCS-serialized encrypted object.
+- `txBytes`: a valid transaction block that satisfies the `seal_approve*` policy function (see [Decryption](#decryption)).
+- `sessionKey`: a session key for the given package (see [Decryption](#decryption)).
+- `allPublicKeys`: an array of Move objects for all public keys in the encryption.
+- `correspondingPublicKeys`: the public keys that correspond to the derived keys. // whats this difference btw allPublicKeys vs correspondingPublicKeys????
```

Comment:
```suggestion
- `correspondingPublicKeys`: the public keys that correspond to the derived keys.
```

Reply:
this the subset of the public keys for decryption, just need to meet the threshold